pegasus/sites.v3/code.org/public/advocacy/state-facts/splat.haml

```diff
@@ -13,6 +13,7 @@ state-facts: true
 %script{src:'https://www.google.com/jsapi'}
 %link{:rel=>'stylesheet', :type=>'text/css', :href=>'/css/state-facts-print.css'}
 %link{:rel=>'stylesheet', :type=>'text/css', :href=>'/css/state-facts.css'}
+%link{:rel=>'stylesheet', :href=>'https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css'}
```

Comment:
Pegasus should already have its own copy of this file...

Reply:
For some reason, without this line the font awesome icons disappear when we generated the PDF

Reply:
Yes, it probably makes sense to add font awesome, but I just think we should refer to our own served version, like we do here: https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites/code.org/views/theme.haml#L7

Reply:
ohhh that makes sense, thanks for pointing that out!